### PR TITLE
Moved salvage magnet values from system to component

### PR DESCRIPTION
**Pull request by dmnct**
## About the PR 
80% less heretical, and salvage magnets are now more easily tunable

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Content.Server/Salvage/SalvageMagnetComponent.cs [hunk 1]
```diff
@@ -47,6 +47,31 @@ public sealed class SalvageMagnetComponent : SharedSalvageMagnetComponent
         [DataField("magnetState")]
         public MagnetState MagnetState = MagnetState.Inactive;
 
+        /// <summary>
+        ///     How long it takes for the magnet to pull in the debris (in seconds)
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("attachingTime")]
+        public int AttachingTime;
```
Comment (0x6273) [inline Content.Server/Salvage/SalvageMagnetComponent.cs:55]: There should be a blank line between all the class fields.

Comment (dmnct) [reply Content.Server/Salvage/SalvageMagnetComponent.cs:55]: Sorry, fixed

**Comment by 0x6273** [comment]
If you are using ints instead of `TimeSpan`s I think the field names should include the unit (in this case seconds). Could you explain why you changed it to ints?

**Comment by dmnct** [comment]
Since the code uses these variables as integers about as often as TimeSpans, I did not see a strong reason to use either type over the other. The decision to go with integer was probably mostly arbitrary, they start as integers in the yml so I went with that

**Pull request merged**